src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulk/scan/transformation/FormAToCaseTransformerTest.java

```diff
@@ -79,7 +79,8 @@ public void shouldTransformFieldsAccordingly() {
             new OcrDataField(OcrFieldName.APPLICANT_FULL_NAME, "Peter Griffin"),
             new OcrDataField(OcrFieldName.RESPONDENT_FULL_NAME, "Louis Griffin"),
             new OcrDataField(OcrFieldName.PROVISION_MADE_FOR, "in connection with matrimonial or civil partnership proceedings"),
-            new OcrDataField(OcrFieldName.NATURE_OF_APPLICATION, "Periodical Payment Order, Pension Attachment Order"),
+            new OcrDataField(OcrFieldName.NATURE_OF_APPLICATION, "a lump sum order,"
```

Comment:
do we need to add more test coverage in here? looks like there's a lot more changed in the main class that had to change here in the test class